tests/publiccloud/run_ltp.pm

```diff
@@ -121,15 +126,67 @@ sub cleanup {
     type_string('', terminate_with => 'ETX');
 
     upload_logs('ltp_log.raw', failok => 1);
-    parse_extra_log(LTP => "$root_dir/ltp_log.json") if (script_run("test -f $root_dir/ltp_log.json") == 0);
-
+    if (script_run("test -f $root_dir/ltp_log.json") == 0) {
+        my $known_issues = download_whitelist();
+        process_ltp_known_issues("$root_dir/ltp_log.json") if $known_issues;
+        parse_extra_log(LTP => "$root_dir/ltp_log.json");
+    }
     if ($self->{my_instance} && script_run("test -f $root_dir/log_instance.sh") == 0) {
         assert_script_run($root_dir . '/log_instance.sh stop ' . $self->instance_log_args());
         assert_script_run("(cd /tmp/log_instance && tar -zcf $root_dir/instance_log.tar.gz *)");
         upload_logs("$root_dir/instance_log.tar.gz", failok => 1);
     }
 }
 
+sub process_ltp_known_issues {
+    my ($self, $ltp_log) = @_;
+    my $decoded_ltp_log = Mojo::JSON::from_json($ltp_log);
+    my $env             = gen_ltp_env();
+    my $suite           = get_required_var('COMMAND_FILE');
+
+    foreach my $test (@{$decoded_ltp_log->{results}}) {
+        my @issues;
+        if (($test->{'status'} eq 'fail' || $test->{'status'} eq 'brok' || $test->{'status'} eq 'warn')) {
```

Comment:
What's in the status field when the test times out and doesn't return anything?

Reply:
I don't know . From what I understand due to this https://github.com/metan-ucw/runltp-ng/blob/master/results.pm#L458 all what I will get will be `fail` . So we need a patch to runltp-ng and than revisit this logic one more time . 
Make sense ? or you think I am missing something ?